### ISSUE TITLE
Success callback

### ISSUE THREAD

**Issue by rickydazla** [Tricky3/campaignmonitorPopUp#2]
@jayvin can you give me an example callback for readme that would hide hide all elements within the `form fieldset` except the `legend` and then change the `legend` text, then removing / hiding the popup after a short period? tx

**Comment by jayvin** [comment]
@rickydazla : you want that callback only inside the CampaignMonitorCallBack.SuccessCallback function right(line 42 in theme.liquid)

**Comment by rickydazla** [comment]
yes

**Comment by jayvin** [comment]
Something like below should be added at line 44.
$('fieldset', wrapper).children().each(function(){
	$(this).is('legend') ?  $(this).text('CUSTOM LEGEND TEXT HERE') : $(this).addClass('hidden')
});

//you will have to force the wrapper(modal) to be visible because it will be hidden by default..
setTimeout(wrapper.addClass('modalize'), 500);

//then hide the wrapper again...
setTimeout(wrapper.removeClass('modalize'), 10000);

**Comment by jayvin** [comment]
By default on successful registration of an email. 

An alert box was being visible previously, after the changes I have done now, you will have either to enable this alert box by setting "ShowAlertBox" to true.

Thing is that, why don't we have a default behavior after a successful callback ? Like the behavior you mentioned above. Of course this can be overwritten by the custom successful callback.

Let me know what you think.

**Comment by rickydazla** [comment]
With campaign monitor previously we have had two broad options 1) redirect to different landing page (specified within CM), or 2) callback. The default callback for CM ajax form is to display an alert. This is the first time I have asked for some kinda generic but different callback. All other implementations we have worked on have involved more customization..

**Comment by rickydazla** [comment]
See: `RedirectOnSubmitSuccess`

**Comment by jayvin** [comment]
Yes I know about that one, you told me that one was being used because client wanted to track signups by counting visits on redirected url. 
Since now we already tracking these signups in google analytics, the question is, should that still be the default callback??

We can keep these current default behaviors but if there more clients that want to show messages like the one we just did, then we can think about it again.

**Comment by rickydazla** [comment]
Well, there will always be a box in CM admin with a place to add a re-direct URL so if that is present then that should be the dominatrix. There might be other tracking pixels that people want to add on that page for Google, Facebook, AdRoll or some other shits that we don't know about...

**Comment by jayvin** [comment]
I updated the default behaviors, see here
https://github.com/Tricky3/twigsHoney/commit/4834455becd0590f8ad93f657e72a89619fded38
, you can merge and deploy it later.
